bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/KeyOnlyFilterAdapter.java

```diff
@@ -32,7 +33,7 @@
  */
 public class KeyOnlyFilterAdapter extends TypedFilterAdapterBase<KeyOnlyFilter> {
   /** Constant <code>TEST_CELL</code> */
-  protected static final Cell TEST_CELL = new KeyValue(
+  protected static final Cell TEST_CELL = new RowCell(
```

Comment:
This is a bit weird...I don't think we should be sending our own subclass as a probe into hbase's impl of transformCell